### ISSUE TITLE
Device.getInfo() on Firefox 109 doesn't show manufacturer and webViewVersion

### ISSUE THREAD

**Issue by de-dan** [ionic-team/capacitor-plugins#1405]
## Bug Report

Device.getInfo() doesn't offer information about manufacturer and webViewVersion on Firefox 109. I checked the web-code and the parsing rules are not applicable for Firefox 109.

### Plugin(s)

Device Plugin

### Capacitor Version

```
@capacitor/cli: 4.6.3
@capacitor/android: 4.6.3
@capacitor/core: 4.6.3
@capacitor/ios: 4.6.3
```
### Platform(s)
Web (Firefox)

### Current Behavior

When I call Device.getInfo() I get the following object:
````
{
  "model": "Macintosh",
  "platform": "web",
  "operatingSystem": "mac",
  "osVersion": "Intel Mac OS X 10.15",
  "manufacturer": "",
  "isVirtual": false,
  "webViewVersion": undefined
}
````

### Expected Behavior

The plugin should give information about manufacturer and webViewVersion as it is available in window.navigator and navigator.userAgent.

**Comment by Ionitron** [comment]
This issue may need more information before it can be addressed. In particular, it will need a reliable Code Reproduction that demonstrates the issue.

Please see the Contributing Guide for [how to create a Code Reproduction](https://github.com/ionic-team/capacitor/blob/HEAD/CONTRIBUTING.md#creating-a-code-reproduction).

Thanks!
Ionitron 💙

**Comment by de-dan** [comment]
The issue is reproducible on every Firefox >108 because the web code you provide isn't valid anymore.
Just call the above mentioned function in that browser. I'm sure you don't need a reproduction repository from me 😅
Best regards
Daniel

**Comment by Ionitron** [comment]
This issue may need more information before it can be addressed. In particular, it will need a reliable Code Reproduction that demonstrates the issue.

Please see the Contributing Guide for [how to create a Code Reproduction](https://github.com/ionic-team/capacitor/blob/HEAD/CONTRIBUTING.md#creating-a-code-reproduction).

Thanks!
Ionitron 💙

**Comment by de-dan** [comment]
Steps to reproduce:

1. Create a new ionic capacitor project
2. Add plugin **Device** 
```
npm install @capacitor/device
npx cap sync
```
3. Call `Device.getInfo()` and log to console, e.g.: `Device.getInfo().then((info) => console.log(info));`
4. Download and install [latest Firefox](https://www.mozilla.org/de/firefox/download/thanks/) (> version 108)
5. Start app in Firefox (you just downloaded)
6. Open developer tools (F12 or right click + inspect)
7. Check properties `manufacturer` and `webViewVersion` in the output.

Nevertheless I created a [repo for you](https://www.youtube.com/watch?v=dQw4w9WgXcQ)

**Comment by jcesarmobile** [comment]
closing due to the lack of reproduction

**Comment by de-dan** [comment]
@jcesarmobile please find here the repo: https://github.com/de-dan/firefoxGT108/tree/main

**Comment by de-dan** [comment]
Dear @jcesarmobile,

I wanted to apologize for my behavior in my previous comment. I now realize that my actions were inappropriate and disrespectful, and I understand why you marked my comments as abuse.

I am sorry for wasting your time and for not following your request to create a reproduction repository. I understand that this would have made it easier for you to identify and fix the issue.

I still believe that the issue I reported is important and would appreciate your help in resolving it. I have created a reproduction repository as you suggested, and have included detailed steps on how to reproduce the error. I hope this will be helpful in resolving the issue.

https://github.com/de-dan/firefoxGT108/

I also proposed a solution for this issue: https://github.com/ionic-team/capacitor-plugins/pull/1419/files

Thank you for your hard work and dedication in maintaining this open-source project.

Sincerely,
Daniel
